### ISSUE TITLE
startbeleg "verloren"

### ISSUE THREAD

**Issue by michael88** [a-sit-plus/at-registrierkassen-mustercode#584]
Ich habe Anfang April für einen Kunden einen gültigen Startbeleg erzeugt und diesen bei Finanz Online eingelesen.

Da ich zuvor Probleme hatte einen gültigen Startbeleg zu erzeugen, habe ich diesen Beleg "händisch" (programmtechnisch gesehen) erzeugt, sprich, noch nicht in der neuen Kassa gespeichert. Die jws-Signatur speicherte ich in einer Txt Datei. Nun, wo der Kunde in den Echtzeitbetrieb gehen will, habe ich den Startbeleg ordnungsgemäss in der datenbank gespeichert und nochmals durch das Prüftool gejagt.

Offensichtlich habe ich damals einen meiner "falschen" Versuche (Umsatzzaehler war nicht 0) gespeichert. Nun habe ich das Problem,
dass ich die Kassa nicht in Betrieb nehmen kann, da der gespeicherte Startbeleg falsch ist bzw. nicht mit dem was ich dem FA übermittelt habe, übereinstimmt.

Frage: Ich habe den Beleg ja mit der Belegcheck App eingelesen. Ist der irgendwo bei Finanz Online gespeichert, so dass ich mir den korrekten Beleg nochmals holen kann oder ist die einzige Möglichkeit, die Kasse stillzulegen und neu anzumelden.  Buchungen wurden noch keine durchgeführt.

**Comment by ErichFreitag** [comment]
Nachdem noch keine Buchungen durchgeführt wurden würde ich nicht lange nachdenken, die alte Kasse stilllegen und korrekt eine neue beginnen. Sie handeln sich sonst wohl mehr Probleme ein als notwendig (Startbeleg als erster Beleg im Lebenszyklus, ...).

**Comment by michael88** [comment]
Ja, das wird wohl das Gescheiteste sein. Ist halt ein bischen peinlich.

**Comment by ErichFreitag** [comment]
Ich finde es nicht peinlich, wenn ein Fehler begangen wurde und der sauber in Ordnung gebracht wird. Wenn noch dazu keine Buchungen stattgefunden haben ist die Ab- und Anmeldung ja keine große Affäre.

**Comment by michael88** [comment]
Danke.
Ich kann alles ,inclusive dem AES Key, gleich lassen, wenn ich der neuen Kassa einen anderen Namen gebe - Korrekt?

**Comment by ErichFreitag** [comment]
Korrekt.
